scripts/cache_warmer/start.py

```diff
@@ -0,0 +1,47 @@
+import os
+import requests
+from dotenv import load_dotenv
+
+load_dotenv("../../.env")
+TX_SERVICE_URL = os.getenv("TRANSACTION_SERVICE_URL")
+PRINT_FORMAT = "{0:<10} {1:>8}::{2:>8}"
+
+
+def get_base_url() -> str:
+    if "staging" not in TX_SERVICE_URL:
+        return "https://safe-client.rinkeby.gnosis.io" if "rinkeby" in TX_SERVICE_URL \
+            else "https://safe-client.mainnet.gnosis.io"
+    else:
+        return "https://safe-client-rinkeby.staging.gnosisdev.com" if "rinkeby" in TX_SERVICE_URL \
+            else "https://safe-client-mainnet.staging.gnosisdev.com"
+
+
+def load_safes() -> list[str]:
+    print("Loading remote ...")
+    top_safes_url = TX_SERVICE_URL + "/api/v1/analytics/multisig-transactions/by-safe/?limit=300"
+    print(top_safes_url)
+    response = requests.get(top_safes_url)
+    return list(map(lambda safe: safe['safe'], response.json()['results']))
```

Comment:
`return [safe['safe'] for safe in response.json()['results']]` would be the preferred way to do that (list comprehension)